akka-actor-tests/src/test/java/akka/actor/ActorCreationTest.java

```diff
@@ -117,5 +141,25 @@ public void testRejectAbstractActorClass() {
       assertEquals(String.format("Actor class [%s] must not be abstract", H.class.getName()), e.getMessage());
     }
   }
-  
+
+  private static Creator<UntypedActor> createAnonymousCreatorInStaticMethod() {
+    return new Creator<UntypedActor>() {
+      @Override
+      public UntypedActor create() throws Exception {
+        return null;
+      }
+    };
+  }
+
+  @Test
+  public void testWrongAnonymousClassCreatedInStaticMethodCreator() {
+    final Creator<UntypedActor> thisIsNotStatic = createAnonymousCreatorInStaticMethod();
```

Comment:
Why would we want to forbid this usage? The point of the check is to tell people that it is not okay to implicitly capture the outer class reference, but there is no such reference in play here. The intention of the ticket was to allow this case after the quoted bug was fixed (which removed the erroneous static flag that made it work previously).